.changeset/@graphql-tools_node-require-7588-dependencies.md

```
@@ -0,0 +1,7 @@
+---
+"@graphql-tools/node-require": patch
+---
+dependencies updates:
+  - Updated dependency [`@graphql-tools/graphql-file-loader@8.1.4` ↗︎](https://www.npmjs.com/package/@graphql-tools/graphql-file-loader/v/8.1.4) (from `8.1.3`, in `dependencies`)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)